### PR TITLE
Add outstanding notes from fortnightly calls

### PR DESCRIPTION
**Pull request by amirmc**
This PR is for the outstanding notes from the last few weeks (added in reverse order).

- [x] 2015-11-11
- [x] 2015-10-28
- [x] 2015-10-14

### CODE REVIEW

**Comment by Drup** [comment]
http://en.99designs.fr/logo-design/contests/make-fun-logo-open-source-unikernel-projects-562612 <- the contest is locked ! I can't see the designs. :(

**Comment by amirmc** [comment]
Yeah, that's because I didn't select the 6 designers to continue working with. :) 

I'll email support and get that phase opened up soon. There are 400 designs there now (the notes say 211 since that was the number at the time). 

Best wishes,
Amir

--
Sent via mobile

**Comment by amirmc** [comment]
The designs are now visible again. We're in the stage where we select 6 designers to continue working with — the site says this has to happen by Saturday but I've tried to explain to the designers that the process may take a little longer for this contest.

**Comment by Drup** [comment]
That's a lot of clouds and a lot of unicorns. x)

**Comment by amirmc** [comment]
That’s probably a result of my brief. I did mention that people had independently come up with the unicorn connection. :)

**Comment by amirmc** [comment]
I'll merge as soon as the CI checks out.

File: tmpl/wiki/weekly/2015-10-14.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+- Functoria
+- Irmin roadmap
+- Pioneer Projects
+- Blog posts and videos
+
+<!-- - OPAM cross-compilation discussion -->
+
+Attendees:
+Daniel Bunzli, Amir Chaudhry (chair), Justin Cormack, Thomas Gazagnaire,
+David Kaloper, Thomas Leonard, Anil Madhavapeddy, Hannes, Menhert,
```
Comment (hannesm) [inline tmpl/wiki/weekly/2015-10-14.md]: ^^ this should be `Hannes Mehnert` (not `Hannes, Menhert` typo + comma)

File: tmpl/wiki/weekly/2015-10-28.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+- Logging
+- Pioneer Projects
+- Blog posts
+- Error handling
+
+
+Attendees:
+Reynir Björnsson, Daniel Bunzli, Amir Chaudhry (chair), Justin Cormack,
+Thomas Gazagnaire, Matt Gray, David Kaloper, Thomas Leonard,
+Anil Madhavapeddy, Hannes, Menhert, Richard (mort) Mortier, Mindy Preston,
```
Comment (hannesm) [inline tmpl/wiki/weekly/2015-10-28.md]: see above...

**Comment by amirmc** [comment]
Sorry @hannesm! I meant to check the spelling and forgot. Will fix before merging.

**Comment by amirmc** [comment]
Fixed the typos in Hannes's surname. Since the previous CI was fine I'm merging now.

**Pull request merged**